src/MoBi.Core/Serialization/Xml/Serializer/MoBiProjectXmlSerializer.cs

```diff
@@ -11,6 +11,7 @@ public override void PerformMapping()
       {
          base.PerformMapping();
          Map(x => x.ReactionDimensionMode);
+         Map(x => x.SimulationSettings);
```

Comment:
This was missed in another issue. MoBi project was created with simulation settings, but wasn't saved to the project and therefore, was missing after loading a project